FreeRTOS-Plus/Demo/coreSNTP_Windows_Simulator/SNTPClientTask.c

```diff
@@ -862,6 +880,9 @@ static void sntpClient_GetTime( SntpTimestamp_t * pCurrentTime )
      * converting from UNIX time to SNTP timestamp. */
     if( ntpSecs > UINT32_MAX )
     {
+        /* Assert when SNTP time rollover. */
+        configASSERT( ntpSecs > UINT32_MAX );
```

Comment:
Same here.

Reply:
The condition is wrong and updated in new commit